build/configs/rtl8721csm/rtl8721csm_download.sh

```diff
@@ -250,6 +251,10 @@ download_specific_partition()
 
 	echo ""
 	echo "Download $exe_name COMPLETE!"
+
+	if [[ "${parts[$partidx]}" == "kernel" ]];then
```

Comment:
When kernel is downloaded, boot parameters should be downloaded also.
That's because boot parameters include kernel information and they can be changed.

Reply:
If this is necessary, it should be done before printing "Download complete".
But this function is only to download specific binary. I think this is not good option to support bootparam in function point of view.
I suggest adding conditional in argument parsing at line 366.
```
case $1 in
	all|ALL)
		download_all
		;;
	erase|ERASE)
		erase $1 $2
		;;
	kernel/KERNEL)
		download_specific_partition $1
		download_specific_partition bootparam
	*)
		download_specific_partition $1
		;;
esac
```